browser/search_engines/tor_window_search_engine_provider_service.cc

```diff
@@ -54,26 +55,14 @@ void TorWindowSearchEngineProviderService::OnTemplateURLServiceChanged() {
 std::unique_ptr<TemplateURLData>
 TorWindowSearchEngineProviderService::GetInitialSearchEngineProvider(
     PrefService* prefs) const {
-  std::unique_ptr<TemplateURLData> provider_data;
-
-  int initial_id = TemplateURLPrepopulateData::GetPrepopulatedDefaultSearch(
```

Comment:
Instead of using `TemplateURLPrepopulateData::duckduckgo` for all regions,
how about using https://github.com/brave/brave-core/pull/9486/files#diff-303c4f4e4e368bfb2681efc54ff35e1412d362740939fc50c7f93b9b92bcca2cL42 for private and tor profile both?

Reply:
We already talked on Slack- but just documenting that we can use regular DDG link for Tor window 👍